fastchat/serve/openai_api_server.py

```diff
@@ -714,7 +714,7 @@ async def create_chat_completion(request: APIChatCompletionRequest):
     if error_check_ret is not None:
         return error_check_ret
 
-    gen_params = get_gen_params(
+    gen_params = await get_gen_params(
```

Comment:
This tripped me up when manually testing.  Seems this API route doesn't get used by many?

Reply:
Yes. The default path does not use this API route. Some contribute it for their special usage.